mobile/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 # android test
-version: 2.0.3+20006
+version: 2.0.4+20008
```

Comment:
updating android version

Reply:
Curious about this android version update. Any considerations for backward compatibility?

Reply:
If I have gotten the question right......Backward compatibility is handled within the application's logic (SqLite database setup, APIs consumed, etc)

Here we are updating two things
1. the version 2.0.4
2. the build number 20008

The build number must always be incremented for every build. So a version can have multiple build numbers. You can think of the version as similar to our docker image versioning though rolling back is close to impossible for mobile apps.

@Baalmart